lmdeploy/model.py

```diff
@@ -125,6 +125,50 @@ def stop_words(self):
         return [45623]
 
 
+@MODELS.register_module(name='llama2')
+class Llama2:
+    """Chat template of LLaMA2 model."""
+
+    def __init__(self):
+
+        B_INST, E_INST = '[INST]', '[/INST]'
+        B_SYS, E_SYS = '<<SYS>>\n', '\n<</SYS>>\n\n'
+
+        DEFAULT_SYSTEM_PROMPT = """
+        You are a helpful, respectful and honest assistant. Always answer as helpfully as possible, while being safe.  Your answers should not include any harmful, unethical, racist, sexist, toxic, dangerous, or illegal content. Please ensure that your responses are socially unbiased and positive in nature.
```

Comment:
There seems to be a extra newline and extra spaces before the system prompt?

Reply:
done